src/pudl/workspace/setup_cli.py

```diff
@@ -102,20 +103,17 @@ def main():
         logfile=args.logfile, loglevel=args.loglevel
     )
 
-    if args.pudl_input:
+    if args.pudl_in:
         pudl_in = pathlib.Path(args.pudl_in).expanduser().resolve()
         if not pathlib.Path.is_dir(pudl_in):
             raise FileNotFoundError(f"Directory not found: {pudl_in}")
-
-    if args.pudl_output:
+        set_path_overrides(input_dir=pudl_in)
```

Comment:
Does using this global envvar override in the CLI have the side effect of changing the value of the environment variable in the user's shell? Is that the desired outcome here?

Reply:
AFAIK the env variables are bound to process. You can inherit from your parent but once new process is started, any changes that would be done to the env would not be propagated back to the parent. Python (and also shell subprocesses) are new processes, hence they should be isolated from the shell that starts them and they should have no impact on the parent's env.

If this was not the case (i.e. it would be easy to accidentally modify shell envs), things would go horribly wrong everywhere all the time IMO.

Reply:
Perhaps this also explains why there was so much emphasis on reverting env variables to former state? I was wondering why are we doing it at all.

Reply:
I think it's the very strange locale / envvar behavior we had with Arelle early on that is making me uneasy here. I don't remember exactly what was going on but it resulted in some kind of persistent change that broke other stuff. Maybe @zschira remembers more...

Reply:
We did a bunch of reverting env vars to former state because they're not automatically cleaned up between tests. Setting them here in the CLI seems fine to me!